script/munin-httpd

```diff
@@ -35,21 +35,21 @@ package main;    ## no critic qw(Modules::ProhibitMultiplePackages)
 
 use Getopt::Long;
 
-my $port = 4948;
+my $listen = ":";
 
 print_usage_and_exit()
   unless GetOptions(
-    "port=i"  => \$port,
+    "listen=s"  => \$listen,
     "version" => \&print_version_and_exit,
     "help"    => \&print_usage_and_exit,
   );
 
-# un-taint port number
-die "ERROR: Invalid port '$port' supplied.\n" unless $port =~ /^(\d+)$/x;
-$port = $1;
+die "ERROR: Invalid listen argument supplied: '$listen'\n" unless $listen =~ /^(\S+)?:(\d+)?$/x;
+my $host = defined $1 ? $1 : "*";
```

Comment:
I would tend to default to localhost here.
(or maybe the empty string, if `Http->new` supports that)

Reply:
"*" was the previous default